feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -87,4 +92,36 @@ class PreviewViewModel @Inject constructor(
             }
         }
     }
+
+    fun startVideoRecording() {
+        Log.d(TAG, "startVideoRecording")
+        recordingJob = viewModelScope.launch {
+
+            try {
+                cameraUseCase.startVideoRecording()
+                _previewUiState.emit(
+                    previewUiState.value.copy(
+                        videoRecordingState = VideoRecordingState.ACTIVE
+                    )
+                )
+                Log.d(TAG, "cameraUseCase.startRecording success")
+            } catch (exception: IllegalStateException) {
+                Log.d(TAG, "cameraUseCase.startVideoRecording error")
+                Log.d(TAG, exception.toString())
+            }
+        }
+    }
+
+    fun stopVideoRecording() {
+        Log.d(TAG, "stopVideoRecording")
+        viewModelScope.launch {
+            _previewUiState.emit(
+                previewUiState.value.copy(
+                    videoRecordingState = VideoRecordingState.INACTIVE
+                )
+            )
+        }
+        cameraUseCase.stopVideoRecording()
```

Comment:
Great ideas. I filed bugs for these under b/281887956

Reply:
I think you're right, we need to implement pausing. I'm not sure how to do that in the UI though, since right now we have the hold to record implementation.

I'll add a bug to track this